src/lib/media.js

```diff
@@ -26,3 +26,7 @@ export const getMediaUrl = (id: string, thumbnail?: boolean) => {
 
   return `http://127.0.0.1:9080/media/original/${id}`;
 };
+
+export const getSvgUri = (svg: string) => {
+  return `data:image/svg+xml;utf8,${svg}`.replace(new RegExp('#', 'g'), '%23')
```

Comment:
Good find! That must have been frustrating. Let's put switching from SVGs to PNGs next on the todo list. I think this is a performance bottleneck in the app and subject to bugs because not all SVGs are equal and some designs can format incorrectly or cause errors.